util/perl/OpenSSL/paramnames.pm

```diff
@@ -173,6 +173,9 @@ my %params = (
     'DIGEST_PARAM_SIZE' =>         "size",         # size_t
     'DIGEST_PARAM_XOF' =>          "xof",          # int, 0 or 1
     'DIGEST_PARAM_ALGID_ABSENT' => "algid-absent", # int, 0 or 1
+    'DIGEST_PARAM_N' =>            "n",            # utf8 string
+    'DIGEST_PARAM_S' =>            "s",            # utf8 string
```

Comment:
I have a slight concern with the param names being just `n` and `s`. Would it be better to use `function-name` and `customization`?

Reply:
Good suggestion,, Changed to these names